pkg/asset/internal/templates.go

```diff
@@ -145,6 +145,7 @@ spec:
         component: kube-apiserver
       annotations:
         checkpointer.alpha.coreos.com/checkpoint: "true"
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
The annotation also exists on `kube-proxy` but not the toleration, so I added it here for consistency. However, daemonsets do respect taints / tolerations so I don't think it hurts to add the toleration to the daemonsets as well. Thoughts?